homeassistant/components/moon/sensor.py

```diff
@@ -13,7 +13,26 @@
 
 DEFAULT_NAME = "Moon"
 
-ICON = "mdi:brightness-3"
+STATE_FIRST_QUARTER = "first_quarter"
+STATE_FULL_MOON = "full_moon"
+STATE_LAST_QUARTER = "last_quarter"
+STATE_NEW_MOON = "new_moon"
+STATE_WANING_CRESCENT = "waning_crescent"
+STATE_WANING_GIBBOUS = "waning_gibbous"
+STATE_WAXING_GIBBOUS = "waxing_gibbous"
+STATE_WAXING_CRESCENT = "waxing_crescent"
+
+MOON_ICONS = {
+    STATE_FIRST_QUARTER: "mdi:moon-first-quarter",
+    STATE_FULL_MOON: "mdi:moon-full",
+    STATE_LAST_QUARTER: "mdi:moon-last-quarter",
+    STATE_NEW_MOON: "mdi:moon-new",
+    STATE_WANING_CRESCENT: "mdi:moon-waning-crescent",   
+    STATE_WANING_GIBBOUS: "mdi:moon-waning-gibbous",
+    STATE_WAXING_CRESCENT: "mdi:moon-waxing-crescent",
+    STATE_WAXING_GIBBOUS: "mdi:moon-waxing-gibbous"
+    
```

Comment:
And remove this blank line, error on Azure DevOps: https://dev.azure.com/home-assistant/Home%20Assistant/_build/results?buildId=16325&view=logs&j=4e214fed-f310-53fa-0018-b38e7beb405b&t=4e7ef364-9046-5187-cd06-98a96bf53c68&l=22

Reply:
thanks!

Reply:
No problem! Thanks for contributing ;-)

Reply:
still 2 tests fail...and it still shows fabaff requested changed. Which I think I made. So don't really understand what's left to do, other than rerun the test?

Reply:
Well not exactly only one test fails which results in the entire build failing. Yes you made the requested changes, now you just have to wait that they are approved. I cannot figure out what the error of the CheckFormat really means, so I would also suggest to rerun it.

Reply:
Ok thanks, I'll wait, and in the mean time was able to run Black locally, which is now saved and tested. Hope this clinches it then! thanks for your support!